src/VisualStudio/CSharp/Test/ProjectSystemShim/CPS/CSharpReferencesTests_CPS.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
+using Microsoft.VisualStudio.LanguageServices.UnitTests.ProjectSystemShim.Framework;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.CSharp.UnitTests.ProjectSystemShim
+{
+    using static CSharpHelpers;
+
+    public partial class CSharpReferenceTests
+    {
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.ProjectSystemShims)]
```

Comment:
I cloned these from existing ProjectShimTests, which also have the same style, I'll retain it for consistency.